.travis.yml

```diff
@@ -0,0 +1,9 @@
+
+language: go
+go: 
+ - tip
+
+script:
+  - gofmt -d -s $(find . -name '*.go')
+  - test -z "$(gofmt -d -s $(find . -name '*.go'))"
+  - go test
```

Comment:
Note that this runs tests only for the top-level package, not for all packages in the repository.
Works for now, as there is only a single package in the repo. We need to be careful to change this when we introduce more packages, or they won't be tested.

The original suggestion (RHMAP-9214) was `go test ./...` that recursively tests all packages.